src/material-experimental/menubar/menubar-item.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {Component, ViewEncapsulation, ChangeDetectionStrategy} from '@angular/core';
+
+/**
+ * A material design MenubarItem adhering to the functionality of CdkMenuItem and
+ * CdkMenuItemTrigger. It's main purpose is to trigger menus and it lives inside of
```

Comment:
Nit: Should this be `its` ? 

```suggestion
 * CdkMenuItemTrigger. Its main purpose is to trigger menus and it lives inside of
```

Reply:
yup, good catch